scripts/jenkins/ci.suse.de/openstack-mediacheck.xml

```diff
@@ -62,14 +61,20 @@
     <hudson.matrix.LabelAxis>
       <name>label</name>
       <values>
-        <string>cloud-trackupstream</string>
+        <string>cloud-trackupstream-sle12</string>
       </values>
     </hudson.matrix.LabelAxis>
   </axes>
-  <combinationFilter>(["Devel:Cloud:3", "Devel:Cloud:4", "Devel:Cloud:5", "Devel:Cloud:5/SLE_12", "Devel:Cloud:6", "Devel:Cloud:6/SLE_12"].contains(project) || subproject == ":")</combinationFilter>
+  <combinationFilter>(["Devel:Cloud:4", "Devel:Cloud:5", "Devel:Cloud:5/SLE_12", "Devel:Cloud:6", "Devel:Cloud:6/SLE_12"].contains(project) || subproject == ":")</combinationFilter>
   <builders>
     <hudson.tasks.Shell>
       <command># temporary, similar to update_automation (not that advanced)
+if [ ! -e  /root/github.com/SUSE/cloud/ ] ; then
+  mkdir -p /root/github.com/SUSE
```

Comment:
I did not write that code, I'm just committing the template that was changed on the server by someone..